### PR TITLE
Undo (#60)

### PR DESCRIPTION
**Pull request by gwenn**

### CODE REVIEW

**Comment by BourgondAries** [comment]
Perhaps these should be squashed (git rebase -i).

**Comment by gwenn** [comment]
This PR depends on this #103 (See Cmd enum introduced there).
Do you suggest that it is possible to create a PR based on the branch of another PR ?

**Comment by BourgondAries** [comment]
I don't know if it's possible, but ss far as I can tell the commits after 5539931 can be squashed into one, and I suggest the commits from #94 be squashed. It's hard to follow this many commits. for a single feature (Vi Mode as I understand it).

**Comment by gwenn** [comment]
Owner/collaborator(s) can choose between merging/squashing/rebasing when closing the PR, no ?

**Comment by BourgondAries** [comment]
Sure, it's just for myself so I can browse the code here instead of pulling your fork, squashing, and then diffing.
